app/assets/stylesheets/styles/pages.scss

```diff
@@ -244,25 +258,25 @@
 
       .material-icons {
         background-color: #42a49a;
+        color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.